### ISSUE TITLE
UnhandledPromiseRejectionWarning: HttpError: No commit found for SHA

### ISSUE THREAD

**Issue by joebowbeer** [slavcodev/coverage-monitor-action#21]
In version 1.1.1, I'm seeing unhandled promise rejection because no commit is found for the given SHA.

The action succeeds in node12, but as the message says it may fail in the future.

## Details

```
Run slavcodev/coverage-monitor-action@1.1.1
(node:2943) UnhandledPromiseRejectionWarning: HttpError: No commit found for SHA: label,MyOrg:Foo,ref,Foo,repo,[object Object],sha,abcde01,user,[object Object]
    at /home/runner/work/_actions/slavcodev/coverage-monitor-action/1.1.1/dist/index.js:13587:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:2943) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2943) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by slavcodev** [comment]
My fault, I did a typo. Already fixed by #22. I also updated the GitHub SDK, so please try v1.2.0.

**Comment by joebowbeer** [comment]
It is fixed in v1.2.0. Thanks!
